main.js

```diff
@@ -1,45 +1,100 @@
-$(document).ready(function(){
-    $('.phone-number').mask('+7(000) 000-00-00');
-    });
+// testing
+    $(document).ready(function(){
+        $('.phone-number').mask('+7(000) 000-00-00');
+        });
+    
 
- let emailValid = /^([a-z0-9_\.-])+@[a-z0-9-]+\.([a-z]{2,4}\.)?[a-z]{2,4}$/;
-		$('#exampleInputEmail').keyup(function(){
-            !emailValid.test($('#exampleInputEmail').val())
-            ? $('#exampleInputEmail').css('border-color' , 'rgb(255, 0, 0)')
-            : $('#exampleInputEmail').css('border-color', '#007d7c');
-		});
+    let inputEmail = document.getElementById('inputEmail');
```

Comment:
please, use const and rename it, userEmailInput sounds much better, fix it in all variables below.